### PR TITLE
Added project Vulnerous by ScorchingShade

### PR DESCRIPTION
**Pull request by ScorchingShade**
Vulnerous is a linux script that aggregates several network security and analysis features into an easy to understand package. Vulnerous provides a simplified commandline interface which is best suited for beginners and security enthusiasts trying to dive into cybersecurity.

Needs new ideas and better ways of implementing the current Vulnerous tool developed exclusively for kali linux.

### CODE REVIEW

**Comment by shiftkey** [comment]
```
2 errors found processing projects:
 - _data/projects/Encryptor.yml:
    - Tag 'AES' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -
 - _data/projects/Vulnerous.yml:
    - Tag 'ShellScript' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -
```

**Comment by ScorchingShade** [comment]
I'll check again

File: _data/projects/Encryptor.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Encrypter
+desc: A password encrypting commandline tool for linux
+site: https://github.com/ScorchingShade/Encryptor
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- python
+- AES
+- encryption
+- beginner
+- issue
```
Comment (ritwik12) [inline _data/projects/Encryptor.yml:10]: @ScorchingShade Please remove this in your next PR, not much-needed tag.

**Comment by ritwik12** [comment]
@ScorchingShade Better if you create individual PRs for each project, would be better to track.

**Pull request closed without merging**